### PR TITLE
[CNXC-202] Fix responsiveness issues with Dicom Viewer

### PR DESCRIPTION
**Pull request by janakitti**
### Problem Statement
The bottom box on the Dicom Viewer page does not scale well. Its height should scale along with its contents.
Screenshot from 2021-03-18 17-12-12

### Implementation
- `em` units were used instead of `vh` to allow for responsiveness when zooming in and out of the page
- `flex` property was used for responsiveness on narrower screens

### CODE REVIEW

**Review by colinywu — APPROVED** [review]
Looks great, confirmed that it fixes the issue of responsiveness and inconsistent spacing with DICOM viewer page

File: src/assets/scss/components/DicomImageViewer.scss [hunk 1]
```diff
@@ -108,14 +108,14 @@ $opacityLevel: 0.75;
 }
 
 .expandedBottom {
+  height: 15em;
```
Comment (boris-nguyen) [inline src/assets/scss/components/DicomImageViewer.scss:111]: I wonder if `13em` would work better? On my display, it seems like there is some extra unnecessary space at the bottom.
Screenshot from 2021-03-23 23-44-36

**Review by boris-nguyen — COMMENTED** [review]
Vertically, the dicom bottom menu is indeed better.

However, responsiveness could be further improved (especially horizontally). I would suggest adding flexbox as well as making the font size dynamic.

Screenshot from 2021-03-23 23-48-52

File: src/components/dicmViewer/dicomViewerBottomBox.tsx [hunk 2]
```diff
@@ -79,34 +79,36 @@ const DicomViewerBottomBox = () => {
           {!isBottomHided ? (<i className="fas fa-angle-down"></i>) : (<i className="fas fa-angle-up"></i>)}
         </span>
       </div>
-      <div className={`flex_row bottomInfoBox ${!isBottomHided ? '' : 'displayNone'}`}>
+      <div className="flex_row">
         <div className="padding-l-2rem">
-          <h2><span><i className="pf-icon pf-icon-info"></i></span></h2>
+            <h2><span><i className="pf-icon pf-icon-info"></i></span></h2>
```
Comment (boris-nguyen) [inline src/components/dicmViewer/dicomViewerBottomBox.tsx:84]:
```suggestion
          <h2><span><i className="pf-icon pf-icon-info"></i></span></h2>
```

**Review by boris-nguyen — APPROVED** [review]
Overall, good job @janakitti 

I would suggest continuously and iteratively improve responsiveness of the Dicom viewer and COVIDNet-UI in general.

As an example, alignment could be further improved when window width is decreased. I would also suggest making the font size responsive so the bottom menu doesnt take over most of the window space.
Screenshot from 2021-04-07 12-19-03


Since originally the scope of this ticket was about the making the height of the Dicom bottom menu responsive, I am approving this PR and we can iteratively work on improving responsiveness for the rest of the portal.

**Pull request merged**